webpack.common.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
     ]
   },
   plugins: [
-    new CleanWebpackPlugin(['dist']),
+    new CleanWebpackPlugin(),
```

Comment:
now the default, as described in https://github.com/johnagan/clean-webpack-plugin/issues/106